From 2e47c6bbbbfcbfd4e2858a0b594b9a1f7bf913ee Mon Sep 17 00:00:00 2001
From: Brandon McAnsh <brandon.mcansh@gmail.com>
Date: Mon, 20 Aug 2018 11:09:26 -0400
Subject: [PATCH 222/249] Track our own elfutils for darwin support around
 features.h

* Needed for https://review.lineageos.org/#/c/LineageOS/android_external_elfutils/+/224679/
* Commit has been submitted to AOSP as well

Change-Id: I78b4ed5f71d390279c7726afc9fbf59e54058331
Signed-off-by: Brandon McAnsh <brandon.mcansh@gmail.com>
---
 default.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default.xml b/default.xml
index d21c97a..ff4cca2 100644
--- a/default.xml
+++ b/default.xml
@@ -118,7 +118,7 @@
   <project path="external/e2fsprogs" name="LineageOS/android_external_e2fsprogs" groups="pdk" />
   <project path="external/easymock" name="platform/external/easymock" groups="pdk" remote="aosp" />
   <project path="external/eigen" name="platform/external/eigen" groups="pdk" remote="aosp" />
-  <project path="external/elfutils" name="platform/external/elfutils" groups="pdk" remote="aosp" />
+  <project path="external/elfutils" name="LineageOS/android_external_elfutils" groups="pdk" />
   <project path="external/emma" name="platform/external/emma" groups="pdk" remote="aosp" />
   <project path="external/error_prone" name="platform/external/error_prone" groups="pdk" remote="aosp" />
   <project path="external/esd" name="platform/external/esd" groups="pdk" remote="aosp" />
-- 
2.17.1

